src/components/DetailsMetadata/DetailsMetadata.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Tooltip from '../../common/Tooltip/Tooltip'
+import TextTooltipTemplate from '../../elements/TooltipTemplate/TextTooltipTemplate'
+import ChipCell from '../../common/ChipCell/ChipCell'
+
+import { generateMetadata } from './detailsMetadata.util'
+
+import './detailsMetadata.scss'
+
+const DetailsMetadata = ({ selectedItem }) => {
+  const { primaryKey } = selectedItem.schema || { primaryKey: '' }
```

Comment:
For default values inc case a variable/property is nullish (`undefined`/`null`) use `??` not `||`
```suggestion
  const { primaryKey } = selectedItem.schema ?? { primaryKey: '' }
```